virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -130,6 +130,7 @@ The SVVP Certification applies to:
 Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
 
 //CNV-26426 [DOCS] Release note: Deprecate TTO
+* The `tekton-tasks-operator` is deprecated and tekton tasks are now deployed by the `ssp-operator`.
```

Comment:
maybe replace tekton tasks with tekton tasks and example pipelines

Reply:
@ksimon1 wdyt^^

Reply:
I agree with suggestion Geetika proposed